frontend/src/store/newUserComparatorState.ts

```diff
@@ -0,0 +1,180 @@
+import { ref } from 'vue';
+import type {
+    FamApplicationUserRoleAssignmentGet,
+    FamUserRoleAssignmentCreate,
+} from 'fam-app-acsctl-api/model';
+import type {
+    FamAccessControlPrivilegeCreateRequest,
+    FamAccessControlPrivilegeGetResponse,
+    FamAppAdminCreateRequest,
+    FamAppAdminGetResponse,
+} from 'fam-admin-mgmt-api/model';
+import type { TabKey } from '@/enum/TabEnum';
+
+interface NewUsersValue {
+    userAccess: FamUserRoleAssignmentCreate[];
+    adminAccess: FamAppAdminCreateRequest[];
+    delegatedAdminAccess: FamAccessControlPrivilegeCreateRequest[];
+}
+
+// Initialize an array to store new users
+const newUsers = ref<NewUsersValue>({
```

Comment:
Hi Jota, I wonder is there any reason that we need to track the new users for all three cases at the same time? I feel each time, there will only have one type of new user access, not sure if we really need an object to store three arrays.

Reply:
It sure can be just one, I'll refactor it